### DISCUSSION TITLE
Allow powerbar front elements to be rendered on top of each other.

### DISCUSSION THREAD

**Discussion by UrsoManso** [ikemen-engine/Ikemen-GO#1773] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

As it is, Ikemen always removes the previous front element when rendering a new one, this feature would allow you to emulate the power bar from MvC2 by rendering on top of the previous one.

### Describe the solution you'd like

A new powerbar parameter that achieves this effect.

### Additional context

![yqa95cwi6ag91 1](https://github.com/ikemen-engine/Ikemen-GO/assets/77592586/91beddf3-7849-4341-8b10-a5376c6265fb)
![Ikemen_GO 2024-05-04 07-45-28](https://github.com/ikemen-engine/Ikemen-GO/assets/77592586/32f6a914-3402-49f4-8d5c-ad8702a7e97e)

**Comment by MUGEN-Slayer** [comment]
You can kind of already do something like this. There is a feature that allows you to change the sprite/animation of bg0 in the same way as changing the front. Like this:

[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/m5cjIIt1wVM/0.jpg)](https://www.youtube.com/watch?v=m5cjIIt1wVM)

**Reply by UrsoManso** [reply]
Which feature is this?

**Reply by MUGEN-Slayer** [reply, marked as answer]
This:

p1.bg0.anim = 3
p1.bg0.layerno = 2
p1.bg01000.anim = 4
p1.bg01000.layerno = 2
p1.bg02000.anim = 5
p1.bg02000.layerno = 2
p1.bg03000.anim = 6
p1.bg03000.layerno = 2

**Reply by UrsoManso** [reply]
Ohhh I see. Thanks.

Btw this is should be commented on the default lifebar. I had no idea these parameters existed.
